bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/api/CatalogQuery.scala

```diff
@@ -34,11 +36,12 @@ case object UCAC4 extends CatalogName("ucac4", "UCAC4 @ Gemini") {
 }
 case object TWOMASS_PSC extends CatalogName("twomass_psc", "TwoMass PSC @ Gemini")
 case object TWOMASS_XSC extends CatalogName("twomass_xsc", "TwoMass XSC @ Gemini")
-case object SIMBAD extends CatalogName("simbad", "Simbad")
+case object SIMBAD extends CatalogName("simbad", "Simbad") {
+  override def checkValidity = false // Simbad sometimes returns non-valid XML, in particular in errors
```

Comment:
Simbad returns the exception and the not found case on a non compliant xml. If we check validity in both cases, they will become errors, which is not correct, only the first case is an actual error